docs/release-notes/17.17.0 and 16.17.0.md

```diff
@@ -0,0 +1,13 @@
+# Model Builder 16.17.0 (VS2019) and 17.17.0 (VS2022) Release
+
+## New Features
+- More settings in advanced training option [Issue 2507](https://github.com/dotnet/machinelearning-modelbuilder/issues/2507)
+- Evaluation code generation for object detection [Issue 2715](https://github.com/dotnet/machinelearning-modelbuilder/issues/2715)
+- Improve training experience on large dataset [Issue 2708](https://github.com/dotnet/machinelearning-modelbuilder/issues/2708)
+- Relative path support for `ModelFilePath` [Issue 2639](https://github.com/dotnet/machinelearning-modelbuilder/issues/2639)
```

Comment:
I think   adding Graph code for evaluation is also a new feature  https://github.com/dotnet/machinelearning-modelbuilder/issues/2470

Reply:
And we also enabled Azure Object detection evaluation dataset pass.